neptunecontrib/viz/parallel_coordinates_plot.py

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+import hiplot as hip
+import neptune
+
+
+def make_parallel_coordinates_plot(columns=None,
```

Comment:
It should be clearly stated that we need to run `neptune.init()` first to know which project it is.
As is it took me a minute to understand it.

It is also missing from the Example (it is the notebook but not here).

Reply:
thanks! I'll add info